src/models/tabularmodel.jl

```diff
@@ -0,0 +1,52 @@
+function emb_sz_rule(n_cat)
```

Comment:
Can we add a comment with a link to where this is taken from?

Reply:
Sure I can add [this link](https://github.com/fastai/fastai/blob/2742fe844573d06e700f869839fb9ec5f3a9bca9/fastai/tabular/model.py#L12). I believe they got this formula experimentally.

Reply:
Can you add it?